src/base-styles/helperClasses/layout/media.css

```diff
@@ -58,3 +58,36 @@ class           | description
   height: var(--media-xl) !important;
   width: var(--media-xl) !important;
 }
+
+/*---
```

Comment:
minor thing - it would be nice to move this into `src/base-styles/helperClasses/layout/display.css`